agent/github_tools.py

```
@@ -542,4 +558,4 @@ async def workspace_commit(owner: str, repo: str, request: Request, body: Worksp
 
         return {"commit": commit_result, "push": push_result, "pr": pr_result}
     except Exception as e:
-        raise HTTPException(status_code=500, detail=str(e))
+        raise HTTPException(status_code=500, detail="Internal server error")
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)